test/parallel/test-repl-mode.js

```diff
@@ -49,17 +49,8 @@ function testStrictMode() {
 
 function testAutoMode() {
   var cli = initRepl(repl.REPL_MODE_MAGIC);
-
-  cli.input.emit('data', `
-    x = 3
-  `.trim() + '\n');
-  assert.equal(cli.output.accumulator.join(''), '> 3\n> ');
-  cli.output.accumulator.length = 0;
-
-  cli.input.emit('data', `
-    let y = 3
-  `.trim() + '\n');
-  assert.equal(cli.output.accumulator.join(''), 'undefined\n> ');
+  assert.equal(cli.output.accumulator.join(''),
+    'magic mode is deprecated. Switched to sloppy mode\n> ');
```

Comment:
If the behaviour should not be affected, why remove existing tests? Thw two ones that were here.